Devscord.DiscordFramework/Services/EmbedMessageSplittingService.cs

```diff
@@ -1,35 +1,31 @@
-﻿using Devscord.DiscordFramework.Middlewares.Contexts;
-using Devscord.DiscordFramework.Services.Factories;
+﻿using Discord;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
 
 namespace Devscord.DiscordFramework.Services
 {
     public class EmbedMessageSplittingService
```

Comment:
nie dawno w trello pojawiło się zadanie z tym związane (numer 264) i jeśli możesz to zrób je.